src/test/isolation2/input/parallel_retrieve_cursor/fault_inject.source

```diff
@@ -249,3 +249,25 @@ SELECT gp_inject_fault('fetch_tuples_from_endpoint', 'resume', dbid)
     WHERE content=2 AND role='p';
 
 DROP TABLE t2;
+
+SELECT gp_inject_fault('alloc_endpoint_slot_full', 'reset', 2);
+SELECT gp_inject_fault('alloc_endpoint_slot_full_reset', 'reset', 2);
+
+-- Test 8: QD shouldn't hang waiting.
+
+DROP TABLE IF EXISTS t3;
```

Comment:
Shouldn't we put this on the ignore block?

Reply:
If some other test that was executed long before yours leaves a t3 table behind, your test will fail.

Reply:
For new code, you can do it right away.